uk.co.compucorp.civicrm.hrcore/CRM/HRCore/SearchTask/SearchTaskAdderInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+interface CRM_HRCore_SearchTask_SearchTaskAdderInterface {
+  /**
+   * Add items to the tasks array in the form:
+   * [
+   *   'title' => <the title to be displayed in the dropdown>,
+   *   'class' => <the class name of the form for the action>
+   * ]
+   *
+   * @param $tasks
+   *
+   * @return void
+   */
+  public static function add(&$tasks);
+
+  /**
+   * Determines whether or not the tasks should be added.
+   *
+   * @param $objectName
+   *
+   * @return bool
+   */
+  public static function shouldAdd($objectName);
+}
```

Comment:
I don't see the point of this interface:
- You're not using actually using it anywhere (you're using the concrete class)
- It only has static methods, so you cannot do something like `__construct(SearchTaskAdderInterface $adder)` and call these methods (because calling static methods from an instance is a deprecated feature in PHP)